dev_setup.sh

```diff
@@ -109,20 +109,19 @@ install_deps() {
     fi
 }
 
-install_deps
+TOP=$(cd $(dirname $0) && pwd -L)
+VIRTUALENV_ROOT=${VIRTUALENV_ROOT:-"${TOP}/.venv"}
+
+install_venv() {
+    python3 -m venv .venv/
```

Comment:
`.venv/` here and below should be `"$VIRTUALENV_ROOT"` if we are still making it configurable.

Reply:
Good catch.

After looking at msm we might want to remove the override for simplicity.